### DISCUSSION TITLE
Connect to remote RaspbeeII adapter via Ser2Net

### DISCUSSION THREAD

**Discussion by razr-bld** [Koenkk/zigbee2mqtt#7246] [Devices]
Hi,

Trying to use Zigbee2Mqtt in a Docker Container, from the docker container I can reach the IP adress of the system with the zigbee device, also I opened the port as described (https://www.zigbee2mqtt.io/how_tos/how_to_connect_to_a_remote_adapter.html)

Edited the yaml configuration file with the line serial:
    port: 'tcp://remote-ip-address:20108'
    
 However Zigbee2Mqtt is unable to connect to the remote zigbee adapter (is is a Raspbee II from Phoscon). Cannot see or find anything about being supported or not)

**Comment by elmarx** [comment]
Hey,
I wanted to do the same and had desperately tried that too, after lots of tries and googling (which took me to this conversation, too), I found it [here](https://forum.iobroker.net/topic/40635/how-to-rasbee-oder-rasbee-ii-via-ser2net) that you need to use `38400` as baud-rate — that's the crucial setting.

The entry in ser2net looks like this now:
```
connection: &con01
    accepter: tcp,20108
    connector: serialdev,/dev/ttyAMA0,38400n81,local
    options:
      kickolduser: true
```

Now starting zigbee2mqtt via docker (with a configuration pointing to the tcp-address (and 'adapter: deconz')) zigbee2mqtt starts and connects to the Raspbee.

Only odd thing so far… it seems ser2net fails after boot and requires a "systemctl restart ser2net", but I'll dig into that later…

**Reply by elmarx** [reply]
OK, I had to add

```
[Unit]
After=network-online.target
Wants=network-online.target
```

to ser2net systemd-configuration (via `systemdctl edit ser2net`).
